src/plone/restapi/services/multilingual/configure.zcml

```diff
@@ -3,6 +3,8 @@
     xmlns:plone="http://namespaces.plone.org/plone"
     xmlns:zcml="http://namespaces.zope.org/zcml">
 
+    <include package="plone.app.multilingual" file="permissions.zcml"/>
```

Comment:
@sneridagh don't we have to add p.a.multilingual to setup.py as well then? I mean at least for Plone 4? Or we wrap it in a conditional zcml...

Reply:
ok, it is already wrapped properly: https://github.com/plone/plone.restapi/blob/55bfaf38c62a041c18ca94b60a703c5eacecbbee/src/plone/restapi/services/configure.zcml#L29

Reply:
@tisto it is already wrapped https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/services/configure.zcml#L29